csr/csr.go

```diff
@@ -49,7 +49,7 @@ type BasicKeyRequest struct {
 
 // NewBasicKeyRequest returns a default BasicKeyRequest.
 func NewBasicKeyRequest() *BasicKeyRequest {
-	return &BasicKeyRequest{"ecdsa", curveP256}
+	return &BasicKeyRequest{"rsa", 2048}
```

Comment:
I still think that if you have "dafaults" command, it should display real defaults